Tests/RegistryTest.php

```diff
@@ -116,36 +116,6 @@ public function testGetUnknownEntityManager()
         $registry->getManager('default');
     }
 
-    public function testResetDefaultEntityManager()
```

Comment:
these two test rely on deprecated behavior, and do not test things in the bundle, be behavior in the doctrine bridge. Since it's not the responsibility of the bundle to test the bridge, they can and should be removed.